pkg/network/http/model.go

```diff
@@ -38,30 +52,57 @@ func (k *httpBatchKey) Prepare(n httpNotification) {
 	k.page_num = C.uint(int(n.batch_idx) % HTTPBatchPages)
 }
 
-// Path returns the URL from the request fragment captured in eBPF with
-// GET variables excluded.
-// Example:
-// For a request fragment "GET /foo?var=bar HTTP/1.1", this method will return "/foo"
-func (tx *httpTX) Path(buffer []byte) []byte {
+func (tx *httpTX) validateFragment(validateHTTPVerb bool, outPath []byte) (bool, []byte) {
 	b := *(*[HTTPBufferSize]byte)(unsafe.Pointer(&tx.request_fragment))
 
-	var i, j int
+	// First, we'll check for the HTTP verb
+	var i int
 	for i = 0; i < len(b) && b[i] != ' '; i++ {
 	}
 
+	if validateHTTPVerb {
+		if _, ok := HTTPVerbs[string(b[:i])]; !ok {
```

Comment:
Is there value in validating the methods/verbs in userspace, when they have to match our eBPF code to be included here?